src/main/java/com/netflix/msl/entityauth/X509AuthenticationData.java

```diff
@@ -42,7 +45,7 @@
  * <p>
  * {@code {
  *   "#mandatory" : [ "x509certificate" ],
```

Comment:
This should become "x509certificate" xor "x509chain" and you need to add "x509chain" and define its contents in the JSON definition and add a <li>.